apps/angular/src/shared/components/header/header.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-toolbar>
+	<div class="header__container container">
+		<div class="header__content">
+			<img src="assets/anime-icon.png" height="60" alt="Anime Catalog App Logo">
```

Comment:
```suggestion
			<img src="assets/anime-icon.png" height="60" alt="">
```

Alt can be left empty if it is a decorative image